index.html

```diff
@@ -23,14 +22,12 @@
       }
 
     </style>
-
   </head>
-  <body>
 
+  <body>
     <video autoplay loop muted>
         <source src="media/best_buddies.mp4" type="video/mp4">
```

Comment:
you can use a conditional message in case video is not supported. look at the example here https://www.w3schools.com/html/html5_video.asp